src/main/java/ca/corefacility/bioinformatics/irida/constraints/ProjectRoleHasCorrectMetadataRole.java

```diff
@@ -13,51 +14,50 @@
 import static java.lang.annotation.RetentionPolicy.RUNTIME;
 
 /**
- * Validation annotation to validate that if the projectRole is set to PROJECT_OWNER
- * for a user/usergroup then the metadataRole must be set to the highest level
- * An array of fields (projectRole, metadataRole) must be supplied
- *
+ * Validation annotation to validate that if the projectRole is set to PROJECT_OWNER for a user/usergroup then the
+ * metadataRole must be set to the highest level An array of fields (projectRole, metadataRole) must be supplied
+ * <p>
  * Example, compare a single user/usergroup projectRole to it's metadataRole:
- * @ProjectRoleHasCorrectMetadataRole(first = "projectRole", second = "metadataRole", message = "server.project.owner.incorrect.metadata.role"))
- *
+ * {@literal @}ProjectRoleHasCorrectMetadataRole(first = "projectRole", second = "metadataRole", message =
+ * "server.project.owner.incorrect.metadata.role"))
  */
-@Target({TYPE, ANNOTATION_TYPE})
+@Target({ TYPE, ANNOTATION_TYPE })
 @Retention(RUNTIME)
 @Constraint(validatedBy = ProjectMetadataRoleValidator.class)
 @Documented
 public @interface ProjectRoleHasCorrectMetadataRole {
-    String message() default "{constraints.projectrolehascorrectmetadatarole}";
-
-    Class<?>[] groups() default {};
-
-    Class<? extends Payload>[] payload() default {};
-
-    /**
-     * @return The projectRole field
-     */
-    String first();
-
-    /**
-     * @return The metadataRole field
-     */
-    String second();
-
-    /**
-     *
-     * Defines several <code>@ProjectRoleHasCorrectMetadataRole</code> annotations on the same element
-     *
-     * @see ProjectRoleHasCorrectMetadataRole
-     *
-     * If a list of validations is required then uncomment the code below
-     *
-     * @Target({TYPE, ANNOTATION_TYPE})
-     * @Retention(RUNTIME)
-     * @Documented
-     * @interface List
-     * {
-     *   ProjectRoleHasCorrectMetadataRole[] value();
-     * }
-     *
-     */
+	String message() default "{constraints.projectrolehascorrectmetadatarole}";
+
+	Class<?>[] groups() default {};
+
+	Class<? extends Payload>[] payload() default {};
+
+	/**
+	 * @return The projectRole field
+	 */
+	String first();
```

Comment:
Can you remove `first` and `second` as we use consistent names for `projectRole` and `metadataRole`

Reply:
Removed in 3c3136c4c1ab2e318d6a50d5a9639281bbdbb16a